src/bun.js/node/zlib/NativeZstd.zig

```diff
@@ -131,7 +128,6 @@ const Context = struct {
     const c = bun.c;
 
     mode: bun.zlib.NodeMode = .NONE,
```

Comment:
Should the default value be removed?

Reply:
NONE is the state before .init is called and after .close is called